src/renderers/web/span-styles.ts

```diff
@@ -169,7 +169,7 @@ export class SpanStyles {
 		}
 		const fontSize = (this._scaleY * fontSizeForLineHeight(this._fontName, this._fontSize * (isTextOnlySpan ? this._fontScaleY : 1), this._fontSizeElement)).toFixed(3);
 		const lineHeight = (this._scaleY * this._fontSize).toFixed(3);
-		span.style.font = `${ fontStyleOrWeight }${ fontSize }px/${ lineHeight }px "${ this._fontName }"`;
+		span.style.font = `${ fontStyleOrWeight }${ fontSize }px/${ lineHeight }px "${ this._fontName }", ${ this._settings.fallbackFonts }`;
```

Comment:
This *seems* to work even if fallbackFonts is empty string, i.e., the browsers I tested seem to be fine with trailing comma and space in the font CSS property, but it is technically wrong.